src/app/components/Link/index.tsx

```diff
@@ -156,23 +152,15 @@ const DesktopLink: FC<DesktopLinkProps> = ({ address, name, to, alwaysTrim, trim
         ) : labelOnly ? (
           <LinkLabel>{trimLongString(address)}</LinkLabel>
         ) : (
-          <MuiLink component={RouterLink} to={to}>
-            {trimLongString(address)}
-          </MuiLink>
+          <RouterLink to={to}>{trimLongString(address)}</RouterLink>
         )}
       </WithHoverHighlighting>
     )
   }
   const label = name ? <HighlightedText text={name} /> : address
   return (
     <WithHoverHighlighting address={address}>
-      {labelOnly ? (
-        <LinkLabel>{label}</LinkLabel>
-      ) : (
-        <MuiLink component={RouterLink} to={to}>
-          {label}
-        </MuiLink>
-      )}
+      {labelOnly ? <LinkLabel>{label}</LinkLabel> : <RouterLink to={to}>{label}</RouterLink>}
```

Comment:
This is not the same as  <MuiLink component={RouterLink} as it's missing styling from Link component, right?